lib/node/runner/api/resource/transaction.go

```diff
@@ -40,3 +40,33 @@ func (t Transaction) Resource() *hal.Resource {
 func (t Transaction) LinkSelf() string {
 	return strings.Replace(URLTransactions, "{id}", t.bt.Hash, -1)
 }
+
+type TransactionStatus struct {
+	Hash   string
+	Status string
+}
+
+func NewTransactionStatus(hash, status string) *TransactionStatus {
```

Comment:
Since both fields are public, and it's only used in one place (`GetTransactionStatusByHashHandler`), I think we could just remove this method and use `TransactionStatus` (or `TxStatus` if you feel the name is too long) directly. It would also remove the memory allocation.